FND/ProductsAndServices/ProductsAndServices.rdf

```diff
@@ -204,7 +208,7 @@
 	<owl:Class rdf:about="&fibo-fnd-pas-pas;Good">
 		<rdfs:label>good</rdfs:label>
 		<owl:disjointWith rdf:resource="&fibo-fnd-acc-cur;AmountOfMoney"/>
-		<owl:disjointWith rdf:resource="&fibo-fnd-plc-loc;RealEstate"/>
+		<owl:disjointWith rdf:resource="&fibo-fnd-plc-rp;RealProperty"/>
```

Comment:
@rivettp - There isn't a relationship, though to avoid unintended consequences of the disjointness, perhaps I should use the parent 'real estate' class instead of real property. I'll make that change.